index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>THEGYM BEM PRACTICE</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <div class="nav__left-side">
+                <div class="nav__logo">Medical Core</div>
+                <div class="nav__items">
+                    <div class="nav__item">Home</div>
+                    <div class="nav__item">Product</div>
+                    <div class="nav__item">Pricing</div>
+                    <div class="nav__item">Contact</div>
+                </div>
+            </div>
+            <div class="nav__menu-icon">
+                <i class="bi bi-list"></i>
+            </div>
+            <div class="nav__right-side">
+                <button class="btn btn--link">Login</button>
+                <button class="btn">JOIN US <i class="bi bi-arrow-right-short"></i></button>
+            </div>
+        </nav>
+
+        <section class="hero-section">
+            <div class="hero-section__title">Welcome</div>
+            <div class="hero-section__intro">A Great Place to Receive Care</div>
+            <div class="hero-section__sub-title">Get your best looking smile now!</div>
+            <div class="hero-section__btn-container">
+                <button class="btn">Join Us</button>
+                <button class="btn btn--outline">Learn more</button>
+            </div>
+        </section>
+    </header>
+
+    <main class="main">
+        <div class="intro-layout">
+            <div class="intro-layout__img-container">
+                <img src="./img/thumb.svg" alt="" class="intro-layout__img">
+            </div>
+            <div class="intro-layout__desc">
+                <div class="intro-layout__border"></div>
+                <div class="intro-layout__title">Meet Our Experts</div>
+                <div class="intro-layout__text">
+                    Problems trying to resolve the conflict between 
+                    the two major realms of Classical physics: 
+                    Newtonian mechanics 
+                </div>
+                <div class="intro-layout__link">Learn more <i class="bi bi-chevron-right"></i></div>
+            </div>
+        </div>
+
+        <div class="team-layout">
+            <div class="team-layout__title">Team</div>
+            <div class="team-layout__big-title">Our Team</div>
```

Comment:
Do not repeat yourself, instead of team-layout__big-title reuse intro-layout__title with modifiers.